admin/sql/updates/20210319-mbs-10647.sql

```diff
@@ -0,0 +1,9 @@
+\set ON_ERROR_STOP 1
+BEGIN;
+
+DROP TRIGGER b_del_label_special ON label;
```

Comment:
(This should exist, but best to avoid failing the upgrade if it doesn't.)
```suggestion
DROP TRIGGER IF EXISTS b_del_label_special ON label;
```